code/style.css

```diff
@@ -112,29 +114,278 @@ main {
+  background-color: #3f93b1;
   color: white;
   border: none;
   border-radius: 4px;
   padding: 16px 40px;
   margin-right: 4px;
   font-size: 16px;
   line-height: 26px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat";
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
+  margin-top: 5px;
 }
 
 button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+footer {
+  text-align: center;
+  padding: 2rem;
+  color: white;
+  width: 100%;
+  margin-top: 13rem;
+  line-height: 1rem;
+}
+
+footer p {
+  font-family: "Quintessential", cursive;
+}
+
+/* MEDIA QUERY – MOBILE – MAX 767 PX*/
+/* -------------------------------------- */
+
+@media (max-width: 767px) {
+  body {
+    margin-bottom: 64px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+    width: 100%;
+  }
+
+  h1 {
+    font-size: 3.1rem;
+    padding: 0.4rem;
+    margin-top: 2.2rem;
+    font-family: "Quintessential", cursive;
+    line-height: 5px;
+  }
+
+  main {
+    height: 410px;
+    padding: 0.9rem;
+  }
+
+  .chat {
+    flex: 1;
+    display: flex;
+    justify-content: flex-start;
+    overflow: scroll;
+    flex-direction: column;
+    padding-bottom: 16px;
+    width: 100%;
+  }
+
+  h2 {
+    font-weight: bold;
+    font-size: 1.8rem;
+    line-height: 2.5rem;
+    color: #fff;
+    text-align: center;
+    margin-bottom: 10px;
+    font-family: "Quintessential", cursive;
+  }
+
+  button {
+    margin-right: 0;
+    font-size: 14px;
+    line-height: 20px;
+    border-radius: 8px;
+    margin-top: 0;
+  }
+
+  .send-btn {
+    font-size: 14px;
+    padding: 12px;
+    margin-right: 0%;
+  }
+
+  #mountainBike {
+    width: 30%;
+    margin-right: 0.5rem;
+    border: none;
+    padding: 10px;
+    text-align: center;
+    line-height: 20px;
+  }
+
+  #ladyBike {
+    width: 30%;
+    margin-right: 0.5rem;
+    border: none;
+    padding: 10px;
+    text-align: center;
+    line-height: 20px;
+  }
+
+  #sportyBike {
+    width: 30%;
+    border: none;
+    padding: 10px;
+    text-align: center;
+    line-height: 20px;
+  }
+
+  #yes {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #no {
+    width: 50%;
+    margin-left: 0.5rem;
+  }
+
+  #chat-form {
+    display: flex;
+    flex-direction: column;
+    margin: 0%;
+    padding: 0%;
+  }
+
+  #yellowColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #purpleColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #blueColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #redColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #greenColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #orangeColor {
+    width: 50%;
+    margin-right: 0.5rem;
+  }
+
+  #happyChoice {
+    width: 50%;
+    margin-right: 0.5rem;
+    padding: 10px;
+  }
+
+  #notHappy {
+    width: 50%;
+    margin-right: 0.5rem;
+    padding: 10px;
+  }
```

Comment:
For the color option buttons here perhaps there could be a class assigned so not to have to repeat the text with the same values.